openedx/features/edly/models.py

```diff
@@ -35,3 +37,20 @@ class Meta:
 
     def __str__(self):
         return '{name}: ({slug})'.format(name=self.name, slug=self.slug)
+
+
+class EdlyUserProfile(models.Model):
+    """
+    EdlyUserProfile model for Edly.
+    """
+    user = models.OneToOneField(User, unique=True, db_index=True, related_name='edly_profile', on_delete=models.CASCADE)
+    edly_sub_organizations = models.ManyToManyField(EdlySubOrganization)
+
+    @property
+    def get_linked_edly_sub_organizations(self):
+        """ Convenience method that returns the user's edly_sub_organization. """
+        sub_org_slugs = []
+        if self.edly_sub_organizations:
+            sub_org_slugs = [sub_org.slug for sub_org in self.edly_sub_organizations.all()]
```

Comment:
we can create a property against this too, i.e. to get all edly sub-organizations slugs.

Reply:
This is already the property returning all edly sub-organizations slugs.

Reply:
We need to add cache of at least 30 minutes in order to avoid any extra queries.

Reply:
This is the model property method. will the Caching model values a good idea? I suggest we look for caching the values where it's used i.e like in the helper method calling this again and again instead of that we can have a Cache set in place.

Reply:
Sure. We will revise this logic in our future optimizations stories.